conf/gconf/cluster.conf.sample

```diff
@@ -0,0 +1,357 @@
+# This is CORTX GConf (Config Store) which serves as input 
+# to all the CORTX Services
+# Location: /etc/cortx/cluster.conf
+
+cortx:
+  external:
+    consul:
+      admin: admin
+      endpoints:
+      - http://consul-server.cortx-cluster.com:80
+      secret: !!binary |
+        MG12NU5BVjBVeWpBSWJuSV9uVjV5UzUxN3lTSVNkNUk1YkRzbjd2QkE9PQ==
+    kafka:
+      admin: admin
+      num_endpoints: 2
+      endpoints:
+      - kafka-server1.cortx-cluster.com
+      - kafka-server2.cortx-cluster.com
+      secret: !!binary |
+        TmRPNThMSm5GOVl5RHgwTVFfaTRJZ3k3YUUwMldmNkdxRE1KYVIxc0E9PQ==
+    openldap:                                                 # deprecated
+      admin: admin                                            # deprecated
+      base_dn: dc=seagate,dc=com                              # deprecated
+      num_endpoints: 2
+      endpoints:                                              # deprecated
+      - ldap://oldap-server.cortx-cluster.com:389 
+      - ssl://oldap-server.cortx-cluster.com:636
+      secret: !!binary |                                      # deprecated
+        cFh1dkw1ZlJ2THR4Rmp1ZnA5Mkt4NndhbGZhWkg0dXdqT0I4bGtxZ3c9PQ==
+      servers:                                                # deprecated
+      - oldap-server1.cortx-cluster.com
+      - oldap-server2.cortx-cluster.com
+      - oldap-server3.cortx-cluster.com
+  common:
+    product_release: LC                             # deprecated
+    security:
+      device_certificate: /etc/cortx/solution/ssl/stx.pem
+      domain_certificate: /etc/cortx/solution/ssl/stx.pem
+      ssl_certificate: /opt/seagate/cortx/s3/install/haproxy/ssl/s3.seagate.com.pem
+    service:
+      admin: admin
+      secret: !!binary |
+        Z0FBQUFBQmhVY1dCdzJ6RkRXT3FSVjJXTWtUM0paZlB1LXFRRW0xNy1rRGhJLVpHUjcxMkVOeDZ6
+        MG9KWmpsM21yTmprZTJkYTBHUU1OWVl4dnZSaVRTeEs4UEdReHhkR1E9PQ==
+    setup_size: small                               # To be removed and derived from limits

```

Comment:
Lets be consistent, mention all old keys as `deprecated`

Reply:
done